addons/hr/models/hr_employee.py

```diff
@@ -870,7 +870,7 @@ def _load_demo_data(self):
         }
         if dep_rd:
             return action_reload
-        convert.convert_file(env=self.env, module='hr', filename='data/scenarios/hr_scenario.xml', idref=None, mode='init', kind="data")
+        convert.convert_file(env=self.sudo().env, module='hr', filename='data/scenarios/hr_scenario.xml', idref=None, mode='init', kind="data")
```

Comment:
Coucou @odoo/rd-security 

Can we have an override?

We need to load the demo data in the xml in sudo when a HR officer click on the `Load sample data` in the no content helper of the `hr.employee` action, as explained in the commit message.

`hr.employee` model is only accessible to the user who has at least hr access and the data inside that file are safe. 🙂 

(maybe there is another solution that I am not aware)

Thanks in advance! 

Have a nice day! 🙂

Reply:
Is it ensured somehow that this cannot be called by a portal user ?

`_load_demo_data` is private because its prefixed with `_ ` so it cannot be called using the xmlrpc api, but how is called `_load_demo_data`, I cannot find ? Because if that method calling `_load_demo_data` can be called through XMLRPC API and there are no access check, adding this sudo gives the possibility to anyone, including portal users, to load the demo data.

Reply:
@beledouxdenis ping 🙂